rpc/src/main/protobuf/exchange.proto

```diff
@@ -3,14 +3,11 @@ syntax = "proto3";
 package kr.ac.postech.paranode.rpc;
 
 service Exchange {
-    
-    rpc SaveRecords (GetMyRecordsRequest) returns (GetMyRecordsReply) {}
+    rpc SaveRecords (SaveRecordsRequest) returns (SaveRecordsReply) {}
 }
 
-message GetMyRecordsRequest {
-    repeated string records = 1;
+message SaveRecordsRequest {
+    repeated bytes records = 1;
 }
 
-message GetMyRecordsReply {
-    bool isNice = 1;
-}
\ No newline at end of file
+message SaveRecordsReply {}
```

Comment:
We mistakenly used 'Request-Reply' instead of the correct 'Request-Response' format defined in our RPC docs. Can you refactor this mistake in the naming?

Reply:
That's my fault during designing. In gRPC, `Request`-`Reply` format is usually used. I confused with HTTP scheme, e.g. `Request`-`Response`.